src/main/java/model/item/Item.java

```diff
@@ -132,36 +136,75 @@ public int getCostAsCopper() {
     }
 
     public int getCostOfOneAsCopper() {
+
+         if (cost.isBlank() || !isTradeable()) {
+             return 0;
+         }
+
         int value = 0;
+        int silverToCopper = Utility.getConfig().getInt("coin.silver.toCopper");
+        int goldToCopper = Utility.getConfig().getInt("coin.gold.toSilver") * silverToCopper;
+        String copper = LanguageUtility.getMessage("coin.copper.short");
+        String silver = LanguageUtility.getMessage("coin.silver.short");
+        String gold = LanguageUtility.getMessage("coin.gold.short");
+
+        List<Character> costList = new ArrayList<>();
+        for (char c : cost.toCharArray()) {
+            costList.add(c);
+        }
+
+        while (!costList.isEmpty()) {
+            int amount = parseNumber(costList);
+            String coin = parseCoin(costList);
+
+            if (coin.equals(copper)) {
+                value += amount;
+            } else if (coin.equals(silver)) {
+                value += amount * silverToCopper;
+            } else if (coin.equals(gold)) {
+                value += amount * goldToCopper;
+            }
+        }
+
+        return value;
+    }
```

Comment:
Don't hate me for this, but I think it would be cleaner to introduce a new Currency object instead of using different integers. This would also simplify any use of currency within the system (you would never need to worry about "is this in copper, is this a string of "10G 10S 5C"? but instead you would have a currency object with toString(), int toCopper() etc.

Reply:
You are right.
Is implemented